src/configuration/routes/https.md

```diff
@@ -0,0 +1,117 @@
+> ```
+> NOW=$(date +"%F_%H:%M:%S") && platform variable:set -W -y force-le-renewal $NOW
+> ```
+> This command sets a [variable](/development/variables.html) for the current branch with the key `force-le-renewal`. The value is the current date and time. You can inspect this variable to know when the last the re-deployment was triggered.
+
+Platform.sh recommends using HTTPS requests for all sites exclusively.  Doing so provides better security, access to certain features that web browsers only permit over HTTPS, and access to HTTP/2 connections on all sites which can greatly improve performance.
+
+How HTTPS redirection is handled depends on the routes you have defined.  Platform.sh recommends specifying all HTTPS routes in your `routes.yaml` file.  That will result in all pages being served over SSL, and any requests for an HTTP URL will automatically be redirected to HTTPS.
+
+```yaml
+"https://{default}/":
+    type: upstream
+    upstream: "app:http"
+
+"https://www.{default}/":
+    type: redirect
+    to: "https://{default}/"
+```
+
+Specifying only HTTP routes will result in duplicate HTTPS routes being created automatically, allowing the site to be served from both HTTP and HTTPS without redirects.
+
+Although Platform.sh does not recommend it, you can also redirect HTTPS requests to HTTP explicitly to serve the site over HTTP only.  The use cases for this configuration are few.
+
+```yaml
+"http://{default}/":
+    type: upstream
+    upstream: "app:http"
+
+"http://www.{default}/":
+    type: redirect
+    to: "http://{default}/"
+
+"https://{default}/":
+    type: redirect
+    to: "http://{default}/"
+
+"https://www.{default}/":
+    type: redirect
+    to: "http://{default}/"
+```
+
+Of course, more complex routing logic is possible if the situation calls for it.  However, we recommend defining HTTPS routes exclusively.
+
+## TLS configuration
+
+Optionally, it's possible to further refine how secure TLS connections are handled on your cluster via the `tls` route property.
+
+```yaml
+https://{default}/:
+    type: upstream
+    upstream: app:http
+    tls:
+        # ...
+```
+
+### `min_version`
+
+Setting a minimum version of TLS will cause the server to automatically reject any connections using an older version of TLS.  While the vast majority of modern browsers will default to TLS 1.2, there are some older browsers that still use insecure older versions of TLS.  Rejecting older versions with known security vulnerabilities is necessary for some security compliance processes.
+
+```yaml
+tls:
+    min_version: TLSv1.2
```

Comment:
Valid values are `TLSv1.0`, `TLSv1.1` and `TLSv1.2`.

Reply:
Updated